src/Pages/jiyeon/Main/Main.js

```diff
@@ -1,13 +1,268 @@
-import React, { Component } from 'react';
+import React from "react";
+import Comment from "./Comment/Comment";
+import "./Main.scss";
 
-class Main extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
+class MainJY extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentInput: "",
+      commentData: [],
+      isBtnActive: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/commentData.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        this.setState({
+          commentData: data,
+        });
+      });
+  }
+
+  handleBtnActive = (event) => {
+    this.setState({
+      isBtnActive: this.state.commentInput.length !== 0,
+      commentInput: event.target.value,
+    });
+  };
+
+  handleCommentAdd = (event) => {
+    event.preventDefault();
+    const { commentInput, commentData } = this.state;
+    if (commentInput.length !== 0) {
```

Comment:
불리안 값을 이용해 조건문을 작성해주셔도 좋겠네요! 
```suggestion
    if (commentInput.length) {
```

Reply:
수정하였습니다!